packages/block-library/src/query/variations.js

```diff
@@ -14,6 +15,27 @@ import {
 } from './icons';
 
 const variations = [
+	{
+		name: '',
+		title: __( 'Posts List' ),
+		icon: postList,
+		attributes: {
+			query: {
+				perPage: 4,
+				pages: 1,
+				offset: 0,
+				postType: 'post',
+				categoryIds: [],
```

Comment:
Yeah, I know but this applies for all blocks with `object` properties. We had discussed that in my PR for this:https://github.com/WordPress/gutenberg/pull/26162, but for back compat it was closed.

>It raises a question of whether all those settings shouldn't be top-level attributes. 

This is something to consider for sure. With my work on Query, I don't think it will have many other properties, so a rework could be considered.